internal/encoding/asn1/ber/common.go

```diff
@@ -13,18 +13,17 @@
 
 package ber
 
-import (
-	"io"
-)
+import "io"
 
-// Writer is the interface that wraps the basic Write and WriteByte methods.
-type Writer interface {
+// writer is the interface that wraps the basic Write and WriteByte methods.
+type writer interface {
 	io.Writer
 	io.ByteWriter
 }
 
 // encodeLength encodes length octets in DER.
 // Reference: ISO/IEC 8825-1: 10.1
+// Reference: https://learn.microsoft.com/windows/win32/seccertenroll/about-encoded-length-and-value-bytes
```

Comment:
It is possible to do
```suggestion
// Reference:
// - ISO/IEC 8825-1: 10.1
// - https://learn.microsoft.com/windows/win32/seccertenroll/about-encoded-length-and-value-bytes
```

Reply:
Updated.